mods/ra/rules/fakes.yaml

```diff
@@ -32,6 +32,42 @@ FPWR:
 	Valued:
 		Cost: 30
 
+TENF:
+	Inherits: ^FakeBuilding
+	Inherits@infiltrate: ^InfiltratableFake
+	Inherits@shape: ^2x2Shape
+	Selectable:
+		Bounds: 48,48
+	Buildable:
+		BuildPaletteOrder: 880
+		Queue: Defense
+		Prerequisites: ~structures.france, ~techlevel.infonly
+		Description: Looks like a Tent.
```

Comment:
This is still referencing Tent. Please change to "Looks like an Allied Barracks." to match the other fake descriptions.

Reply:
👍